packages/api/src/external/carequality/document/shared.ts

```diff
@@ -21,12 +21,16 @@ export function containsMetriportId(
   return docRef.metriportId != undefined;
 }
 
-export function filterDocRefsWithMetriportId(
-  documentReferences: DocumentReference[]
```

Comment:
we already have a function called `containsMetriportId` that does this. Not nessecary to have this one too